server/pages/badge.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+const BranchName = ( { branchName, commitChecksum } ) => (
+	<span className={ 'environment branch-name' } title={ 'Commit ' + commitChecksum }>
+		{ branchName }
+	</span>
+);
+
+const DevDocsLink = ( { devDocsUrl, docs } ) => (
```

Comment:
The docs link seems to have disappeared:

![image](https://cloud.githubusercontent.com/assets/1779930/19192611/c87c41ea-8c74-11e6-9465-c8779fe9c56a.png)

(Should be to the right of branch name)